site/docs/try/releases.md

```diff
@@ -1,5 +1,9 @@
 # Releases
 
+## 0.18.0 Release (January 13, 2022)
+
+* Uses commit-timestamp "now" for merged/transplanted commits
```

Comment:
shoudln't this have more?

Reply:
Argh! Seems I swapped the release notes files